homeassistant/components/sabnzbd.py

```diff
@@ -102,7 +102,7 @@
                                         hass.config.path(CONFIG_FILE))
         api_key = conf.get(base_url, {}).get(CONF_API_KEY, '')
 
-    sab_api = SabnzbdApi(base_url, api_key)
+    sab_api = SabnzbdApi(base_url, api_key, "sabnzbd")
```

Comment:
Is it possible to configure the web root in the SABnzbd configuration? If yes, then it should become a configuration variable which defaults to `sabnzbd`.

Reply:
I was going to propose that as well. It is possible but cannot be done by Sabnzbd itself. You need to make a rule in apache/nginx. See: https://sabnzbd.org/wiki/extra/howto-apache. In that case, however, the original webroot is still accessible inside the netword however the option may still be useful if accessing it behind a reverse proxy.

I can submit a new PR with the option included

Reply:
Would be simple to include it here. As the change is about the same thing.